generators/app/templates/.github/workflows/_ci.yml

```diff
@@ -0,0 +1,77 @@
+name: ci
```

Comment:
do we want this here? Since it's sort of specific to Videojs projects only and not everyone may want this. Plus, it'll be yet another place we have to maintain this file.

Reply:
Maybe, but we already included the travis config previously and didn't add it to the project if it was closed source. Should we fetch it on install?

Reply:
downloading it from the .github repo sounds like a good idea. That way we don't have to maintain the script in yet another location.

Reply:
ok I have it downloading now.